lib/utils/testLauncherHelper.js

```diff
@@ -123,14 +124,18 @@ function followChildProcess(child, callback) {
 
 /**
  * Validate test launcher's args based on session type
- * @param {String} type session type "AUTOMATED" or "INTERACTIVE"
+ * @param {String} type session type "AUTOMATED" or "INTERACTIVE", "TEST_COMMAND"
```

Comment:
AUTOMATED and INTERACTIVE should be in lower case?

Reply:
no, it is provided as `AUTOMATED.toUpperCase()`